lib/compat/wordpress-6.5/blocks.php

```diff
@@ -167,7 +167,11 @@ function gutenberg_process_block_bindings( $block_content, $block, $block_instan
 	);
 
 	// If the block doesn't have the bindings property or isn't one of the allowed block types, return.
-	if ( ! isset( $block['attrs']['metadata']['bindings'] ) || ! isset( $allowed_blocks[ $block_instance->name ] ) ) {
+	if (
+		! isset( $allowed_blocks[ $block_instance->name ] ) ||
+		empty( $parsed_block['attrs']['metadata']['bindings'] ) ||
```

Comment:
[Here](https://github.com/WordPress/wordpress-develop/pull/5888/commits/604f7a2cb3767163c7d12a4f571f76b7a47d20a6#diff-c569b1ecb11a007a563788a481710b51e0cb21cd1d1885490bf9c91777842f23R249)
we added a check with `isset()` but I think `empty()` is even more appropriate
(like you did). Let's remember to update it in Core

Reply:
It was included in this backport: [link](https://github.com/WordPress/wordpress-develop/pull/6016/files#diff-c569b1ecb11a007a563788a481710b51e0cb21cd1d1885490bf9c91777842f23R249-R253). From what I can see, it seems to be already in core: [link](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/class-wp-block.php#L249-L253).